gef.py

```diff
@@ -2951,6 +2956,32 @@ def exit_handler(event):
         __gef_remote__ = None
     return
 
+def load_libc_args():
+    # load libc function arguments' definitions
+    if not get_gef_setting("context.libc_args"):
+        return
+
+    path = get_gef_setting("context.libc_args_path")
+    if path is None:
+        err("Settings context.libc_args_path not set but context.libc_args is True. Make sure you have gef-extras installed")
+        return
+
+    _arch_mode = current_arch.arch.lower() + '_' + current_arch.mode
+    _libc_args_file = "{}/{}.json".format(path, _arch_mode)
+
+    global libc_args_definitions
+
+    # current arch and mode already loaded
+    if _arch_mode in libc_args_definitions and len(libc_args_definitions[_arch_mode]):
+        return
+
+    try:
+        libc_args_definitions[_arch_mode] = {}
+        with open(_libc_args_file) as _libc_args:
+            libc_args_definitions[_arch_mode] = json.load(_libc_args)
+    except:
```

Comment:
Catch an exception type.

We probably want to handle different exceptions, for example, file not found, un parsable, etc.

Reply:
e.g. `OSError` vs `json.decoder.JSONDecodeError`

Reply:
handled two cases: file missing (both directory and file), file malformed